sites/public/styles/overrides.scss

```diff
@@ -99,8 +115,11 @@
             @screen md {
               // Negative margin to center around the shield
               margin: -32px 0 0;
+              max-height: 6rem;
```

Comment:
This was min-height before?

Reply:
Yeah I removed that because it wasn't doing anything. The image is larger than the max-height so that's the relevant css. These probably all should just be height but I didn't want to change things too much here.